src/napari_workflows/_tests/test_function.py

```diff
@@ -57,7 +57,7 @@ def refine(image:napari.types.LabelsData) -> napari.types.LabelsData:
     gui = make_gui(segment, viewer, auto_call=True)
     viewer.window.add_dock_widget(gui)
     # invoke execution / workflow update
-    gui.sigma.value = gui.sigma.value
+    gui.sigma.value = 5.1
```

Comment:
apparently, the  workflow update is only triggered if the value actually changes